### ISSUE TITLE
Drop `BxmAvO` and Add VFATX vs. VFATY Mismatch Flags in FW & Event Format

### ISSUE THREAD

**Issue by bdorney** [evka85/GEM_AMC#36]
Following discussion on [Slides 2-5](https://indico.cern.ch/event/820580/contributions/3430007/attachments/1846499/3029677/BDorney_QC8_20190517.pdf) would like to drop `BxmAvO` from the GEM Event Data format and any corresponding status bit/flag registers in the FW.

Propose to add `BxmVvV` to represent BX mismatch VFATX vs VFATY where for all `X != Y` a check on that VFATX BX is made and compared against VFATY BX.  If this is not true a flag is raised.

Also add a FW flag `AMC_SEES_VFATX_VFATY_BX_MISMATCH` which would be raised high if a two or more VFATs had mismatching BX's.  Then in the `DAQ.STATUS` node I would propose to add a `LAST_VFAT_BX_SENT` register which for each L1A would store the last BX that was sent by the VFAT so if `AMC_SEES_VFATX_VFATY_BX_MISMATCH` was raised high you could check which VFATs were showing the mismatch.

@jsturdy @evka85 @mexanick

**Comment by evka85** [comment]
implemented in v3.8.4
VFAT-VFAT mixed BC replaced the BxmAvO
VFAT-VFAT mixed EC replaced the OOScAvO

The BC (and also EC) mismatch flag registers already exist: VFAT_MIXED_BC and VFAT_MIXED_BC
The exact VFATs that mismatch can be found out from the data. I'm not sure it's worth trying to implement this in the registers, because once there is a mismatch, typically all following events are also mixed up, and also there's an ambiguity in determining which VFATs have wrong BC and which ones have the right one. So I hope we can close this issue, at least for now and see how it goes.

**Comment by bdorney** [comment]
Yeah as long as the flag goes high when this happens this is really all that matters.  The idea of a "correct BX" in this case is probably meaningless :D

Closing now; if additional changes to functionality are needed I'll open a new issue.  Thanks!
